test/user.test.js

```diff
@@ -1828,17 +1760,12 @@ describe('User', function() {
         User.afterRemote('create', function(ctx, user, next) {
           assert(user, 'afterRemote should include result');
 
-          var options = {
-            type: 'email',
-            to: user.email,
-            from: 'noreply@myapp.org',
-            redirect: '/',
+          Object.assign(verifyOptions, {
             host: 'myapp.org',
-            port: 3000,
```

Comment:
I think it's better to keep explicit `port: 3000`, because we are asserting this port number few lines below (`http://myapp.org:3000/test-users/confirm`).

When the default port is used, it's not clear to readers of this test where is the port 3000 coming from.